src/Compute/computeEffect.ts

```diff
@@ -287,13 +289,13 @@ export class ComputeEffect {
         }
 
         // Is in local store ?
-        if (ShaderStore.ShadersStore[shader + key + "Shader"]) {
-            callback(ShaderStore.ShadersStore[shader + key + "Shader"]);
+        if (ShaderStore.GetShadersStore(this._shaderLanguage)[shader + key + "Shader"]) {

```

Comment:
should this be stored instead of calling every time ?

Reply:
Done